src/test/java/org/assertj/core/api/WithAssertions_delegation_Test.java

```diff
@@ -385,15 +380,15 @@ public void withAssertions_assertThat_iterable_assert_class_Test() {
    */
   @Test
   public void withAssertions_assertThat_iterable_assert_factory_Test() {
-    assertThat((Iterable<TestItem>) Arrays.asList(ITEMS), t -> new ObjectAssert<>(t)).first().isEqualTo(ITEMS[0]);
+    assertThat((Iterable<TestItem>) Arrays.asList(ITEMS), ObjectAssert::new).first().isEqualTo(ITEMS[0]);
   }
 
   /**
    * Test that the delegate method is called.
    */
   @Test
   public void withAssertions_assertThat_iterator_Test() {
-    assertThat(list(ITEMS).iterator()).hasNext();
+    assertThat(Lists.list(ITEMS).iterator()).hasNext();
```

Comment:
let's keep the static import for (slightly) better readability

Reply:
This test class implements `WithAssertions` which extends `InstanceOfAssertFactories`, so now in the scope we have `InstanceOfAssertFactories.list()` which is clashing with `Lists.list()`